cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/DurableLogRecoveryCommand.java

```diff
@@ -174,7 +174,6 @@ private void flushToStorage(Map<Integer, DebugStreamSegmentContainer> debugStrea
     private void stopDebugSegmentContainers(Map<Integer, DebugStreamSegmentContainer> debugStreamSegmentContainerMap)
             throws Exception {
         for (val debugSegmentContainerEntry : debugStreamSegmentContainerMap.entrySet()) {
-            Services.stopAsync(debugSegmentContainerEntry.getValue(), executorService).get(TIMEOUT.toMillis(), TimeUnit.MILLISECONDS);
```

Comment:
As you can see [here](https://github.com/pravega/pravega/issues/5481#issuecomment-762972854), the test sometimes fails at this line. 
Further down, we already close the debug segment containers, so this line is not required. So, removing it.